blockchain/state/doc.go

```diff
@@ -18,5 +18,26 @@
 // This file is derived from core/state/statdb.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package state provides a caching layer atop the Klaytn state trie.
+/*
+Package state provides a caching layer atop the Klaytn state trie.
+This package is used to read/write stateObject from/to StateDB and it also acts as a bridge between
+the objects and the persistent layer.
+
+Overview of state package
+
+stateObject represents a Klaytn account which is being modified. It is identified by its address.
```

Comment:
The expression, `which is being modified`, looks little ambiguous for me. 
How about `stateObject represents the state of a Klayn account at a specific block number. `

Reply:
@aidan-kwon `at a specific block number` is not exactly what it is. Since a stateObject can represent the intermediate state (or state transition) between blocks. How about just removing `which is being modified`?

Reply:
Good suggestion. 
It can be combined with the next line.
`stateObject represents a Klaytn account identified by its address.`